Jorge/HTML & CSS/index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>First Task</title>
+</head>
+<body>
+  <div class="main-container">
+    <nav class="header">
+      <div class="header__logo">
+        <img src="images/broly.png" alt="logo">
+      </div>
+      <div class="header__menu">
+        <ul>
+          <li><a href="#">FEATURES</a></li>
+          <li><a href="#">TEMPLATES</a></li>
+          <li><a href="#">INTEGRATION</a></li>
+          <li><a href="#">PRICING</a></li>
+        </ul>
+      </div>
+      <div class="header__button">
+        <button class="button">SIGN IN</button>
+        <button class="button primary">SIGN UP</button>
+      </div>
+    </nav>
+    <div class="hero">
+      <h3>Hola, me llamo</h3>
+      <h1>Jorge</h1>
```

Comment:
lets try to put these 2 lines in a h1 with a span, and use a css reseter to style the font size according to your needs